mlperf_logging/compliance_checker/mlp_compliance.py

```diff
@@ -76,12 +76,13 @@ def overwrite_messages(self, keys):
 
     def log_messages(self):
         message_separator = '\n' + '-' * 30 + '\n'
-
-        print(message_separator.join([
-            *self.warnings.values(),
-            *self.overwritable.values(),
-            *self.not_overwritable
-        ]))
+        message = message_separator.join([
+                    *self.warnings.values(),
+                    *self.overwritable.values(),
+                    *self.not_overwritable
+        ])
+        if message:
+          print(message)
```

Comment:
@bitfort @xyhuang In the future, can we avoid such changes in the last days before the deadline? Can we wait with it until v0.8?

These are just cosmetic changes, while our post-processing scripts need alignment to such changes because the output format was changed.